<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventClass</class>
 <widget class="QDialog" name="AddEventClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar Evento</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Identificador</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idEvent">
        <property name="toolTip">
         <string>Cargue el nombre del identificador del evento</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tipo de Evento</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="teclaOption">
        <property name="text">
         <string>Simulación de Tecla</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="appOption">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Ejecución de aplicación</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Simulación de Tecla</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>226</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>Tecla</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>69</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>69</width>
          <height>20</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>Up</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Down</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Esc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Space</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Intro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tab</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pg.Up</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pg.Dn</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alt+Tab</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>225</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Ejecución de aplicación</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Aplicación o Archivo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="appEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="findApp">
        <property name="toolTip">
         <string>Seleccione la aplicación a ejecutar</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Argumentos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="argsEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="argCheck">
        <property name="text">
         <string>¿Argumentos?</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="changeButton">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acceptButton">
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>teclaOption</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeButton</sender>
   <signal>clicked()</signal>
   <receiver>AddEventClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>appOption</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>argCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>argsEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>appOption</sender>
   <signal>toggled(bool)</signal>
   <receiver>appEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>appOption</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>appOption</sender>
   <signal>toggled(bool)</signal>
   <receiver>argsEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findApp</sender>
   <signal>clicked()</signal>
   <receiver>AddEventClass</receiver>
   <slot>openExecutableFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>appOption</sender>
   <signal>toggled(bool)</signal>
   <receiver>findApp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>AddEventClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openExecutableFile()</slot>
 </slots>
</ui>
